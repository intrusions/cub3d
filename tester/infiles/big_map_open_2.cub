NO ./path_to_the_north_texture

SO ./path_to_the_south_texture

WE ./path_to_the_west_texture

EA ./path_to_the_east_texture

F 220,100,0

C 225,30,0
111111111111111111111111111111111111111111111111111111111111111111111111
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
1000000000110000000000000001100000000000000000000011000000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000000000000000000000000000000000000000000000000000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
110000011101010100000000000000000000000000000000000000001111011110N0111
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
10000000001100000000000000011000000000000000000000110000000000000000001
11111111111111111111111111111111111111111111111111111111111111111111111
11111111111111111111111111111111111111111111111111111111111111111111111