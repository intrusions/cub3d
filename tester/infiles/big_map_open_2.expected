Loading scene...
Loading map...
Error
	big_map_open_2.cub: Map: is open
Quitting...
